spring-cloud-kubernetes-commons/pom.xml

```diff
@@ -50,13 +50,29 @@
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-test</artifactId>
+			<exclusions>
+				<exclusion>
+					<groupId>org.mockito</groupId>
+					<artifactId>mockito-core</artifactId>
+				</exclusion>
+			</exclusions>
 			<scope>test</scope>
 		</dependency>
 		<dependency>
 			<groupId>org.junit.vintage</groupId>
 			<artifactId>junit-vintage-engine</artifactId>
 			<scope>test</scope>
 		</dependency>
+
+		<dependency>
```

Comment:
need `mockito-inline` because of the need for `Mockito::mockStatic`, which in turn is needed to be able to mock `Paths::get` and `Files::readAllBytes/isRegularFile`